lib/static-methods.js

```diff
@@ -341,7 +341,7 @@ exports.reduce = async (array, callback, initialValue) => {
   }
   let i;
   let previousValue;
-  if (initialValue) {
+  if (typeof initialValue !== 'undefined') {
```

Comment:
Why did you prefer `typeof` over `initialValue !== undefined`?

I think in our case the latter would be fine as well, since `initialValue` is always declared (it's an argument) and `undefined` can't be reassigned since ES5.